integration/aggregate_documents_compat_test.go

```diff
@@ -579,6 +689,7 @@ func TestAggregateCompatGroup(t *testing.T) {
 				{"_id", "$$s"},
 			}}}},
 			resultType: emptyResult,
+			skip:       "https://github.com/FerretDB/FerretDB/issues/2275",
```

Comment:
It's better to return `not implemented`, than `undefined variable` until we implement variables.